torchbiggraph/converters/export_to_tsv.py

```diff
@@ -26,9 +25,9 @@ def write(outf: TextIO, key: Iterable[str], value: Iterable[float]) -> None:
 
 
 def make_tsv(
-    config: ConfigSchema,
-    entities_tf: TextIO,
-    relation_types_tf: TextIO,
+        config: ConfigSchema,
+        entities_tf: TextIO,
+        relation_types_tf: TextIO,
```

Comment:
This extra indentation is inconsistent with our style, which is Black-inspired (to some extent). Could you remove it and leave it as it was? To see how Black would format something you can use https://black.now.sh/.